app/routes/people/person/personal-information.js

```diff
@@ -1,5 +1,11 @@
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
+import { assert } from '@ember/debug';
+import {
+  findPrimaryContact,
+  findSecondaryContact,
+} from '../../../models/contact-point';
```

Comment:
It's recommended to use absolute paths instead of relative paths since that makes it easier to move things around.

```suggestion
} from 'frontend-contact-hub/models/contact-point';
```

VS Code can be configured to understand these paths by adding a jsconfig file to the root of the project:

```js
{
  "compilerOptions": {
    "baseUrl": ".",
    "experimentalDecorators": true,
    "paths": {
      "frontend-contact-hub/*": ["./app/*"],
      "frontend-contact-hub/config/environment": ["./app/config/environment"],
      "frontend-contact-hub/tests/*": ["./tests/*"]
    },
    "target": "es6"
  },
  "exclude": [".git", "coverage", "dist", "node_modules", "vendor"]
}
```
It should then generate the correct import paths.